src/main/java/com/hyperwallet/clientsdk/util/HyperwalletMultipartUtils.java

```diff
@@ -0,0 +1,72 @@
+package com.hyperwallet.clientsdk.util;
+
+import com.hyperwallet.clientsdk.HyperwalletException;
+import com.hyperwallet.clientsdk.model.HyperwalletVerificationDocument;
+import net.minidev.json.JSONObject;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class HyperwalletMultipartUtils {
+
+    public static Multipart convert(List<HyperwalletVerificationDocument> uploadList) throws IOException {
+
+        final String crlf = "\r\n";
```

Comment:
Can it be used from MultiPartRequest? Seems duplicated

Reply:
its a private in MultipartRequest